jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ForwardedRequestCustomizer.java

```diff
@@ -872,7 +877,7 @@ private MutableHostPort getFor()
          */
         public void handleCipherSuite(HttpField field)
         {
-            _request.setAttribute("jakarta.servlet.request.cipher_suite", field.getValue());
+            _sslSessionData = EndPoint.SslSessionData.from(_sslSessionData, null, null, field.getValue(), null);
```

Comment:
As this PR moved the class (record?) to `EndPoint` we can't have separate classes for how the information was provided.  Besides, the whole point is that the user of this should not know if the data comes from a connection or a proxy or the proxy protocol.

However, I agree it is a little strange having the SslSession as part of the SslSession data, specially when this is a record.     But every attribute we add has a cost, so putting multiple related values behind a single attribute has value.
I'll try converting the record to a class and hopefully with less null values passed it will look better?